### PR TITLE
Updated Dockerfile with JANUS_VERSION=v1.0.0

### PR DESCRIPTION
**Pull request by jhon-freitez-jemmic**
Replaced the value of the variable `JANUS_VERSION` from `v0.10.3` to `v1.0.0` on the `Dockerfile`.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ ENV CONFIG_PATH="/opt/janus/etc/janus"
 # docker build arguments
 ARG BUILD_SRC="/usr/local/src"
 
-ARG JANUS_VERSION="v0.10.3"
+ARG JANUS_VERSION="v1.0.0"
 ARG JANUS_LIBNICE_VERSION="0.1.17"
```
Comment (christophe-kamphaus-jemmic) [inline Dockerfile]: @jhon-freitez-jemmic Also update the other dependencies.
For example libnice here is still version 0.1.17, while on https://github.com/meetecho/janus-gateway at least 0.1.18 is recommended.
Also check if any other changes in dependencies are needed (e.g. new dependencies).

File: Dockerfile [hunk 2]
```diff
@@ -17,13 +17,13 @@ ENV CONFIG_PATH="/opt/janus/etc/janus"
...
 ARG JANUS_PAHO_MQTT_VERSION="v1.3.4"
-ARG JANUS_RABBITMQ_VERSION="v0.10.0"
+ARG JANUS_RABBITMQ_VERSION="v1.0.4"
 ARG JANUS_LIBSRTP_VERSION="2.3.0"
-ARG JANUS_USRSCTP_VERSION="547d3b46c64876c0336b9eef297fda58dbe1adaf"
-ARG JANUS_BORINGSSL_VERSION="8b601c88fb9b82b49407b018ff5e626a59600669"
-ARG JANUS_LIBWEBSOCKETS_VERSION="v4.0.20"
+ARG JANUS_USRSCTP_VERSION="0.9.5.0"
+ARG JANUS_BORINGSSL_VERSION="ac3f4fb8e1e3fd6da51033514e247e7924b16439"
+ARG JANUS_LIBWEBSOCKETS_VERSION="v4.3.0"
```
Comment (christophe-kamphaus-jemmic) [inline Dockerfile]: Why not use 4.3.1?

File: Dockerfile [hunk 3]
```diff
@@ -17,13 +17,13 @@ ENV CONFIG_PATH="/opt/janus/etc/janus"
 ARG BUILD_SRC="/usr/local/src"
 
 ARG JANUS_VERSION="v1.0.0"
-ARG JANUS_LIBNICE_VERSION="0.1.17"
+ARG JANUS_LIBNICE_VERSION="0.1.18"
 ARG JANUS_PAHO_MQTT_VERSION="v1.3.4"
-ARG JANUS_RABBITMQ_VERSION="v0.10.0"
+ARG JANUS_RABBITMQ_VERSION="v1.0.4"
 ARG JANUS_LIBSRTP_VERSION="2.3.0"
```
Comment (christophe-kamphaus-jemmic) [inline Dockerfile]: Why not update to `2.4.2`?

File: Dockerfile [hunk 4]
```diff
@@ -17,13 +17,13 @@ ENV CONFIG_PATH="/opt/janus/etc/janus"
 ARG BUILD_SRC="/usr/local/src"
 
 ARG JANUS_VERSION="v1.0.0"
-ARG JANUS_LIBNICE_VERSION="0.1.17"
+ARG JANUS_LIBNICE_VERSION="0.1.18"
 ARG JANUS_PAHO_MQTT_VERSION="v1.3.4"
-ARG JANUS_RABBITMQ_VERSION="v0.10.0"
+ARG JANUS_RABBITMQ_VERSION="v1.0.4"
```
Comment (christophe-kamphaus-jemmic) [inline Dockerfile]: Are you sure this tag exists?
When checking https://github.com/alanxz/rabbitmq-c I see that v0.11.0 is the latest tag.

File: Dockerfile [hunk 5]
```diff
@@ -17,13 +17,13 @@ ENV CONFIG_PATH="/opt/janus/etc/janus"
 ARG BUILD_SRC="/usr/local/src"
 
 ARG JANUS_VERSION="v1.0.0"
-ARG JANUS_LIBNICE_VERSION="0.1.17"
+ARG JANUS_LIBNICE_VERSION="0.1.18"
 ARG JANUS_PAHO_MQTT_VERSION="v1.3.4"
```
Comment (christophe-kamphaus-jemmic) [inline Dockerfile]: Why not update to v1.3.10?
https://github.com/eclipse/paho.mqtt.c/releases

File: Dockerfile [hunk 6]
```diff
@@ -17,13 +17,13 @@ ENV CONFIG_PATH="/opt/janus/etc/janus"
...
-ARG JANUS_RABBITMQ_VERSION="v0.10.0"
+ARG JANUS_RABBITMQ_VERSION="v1.0.4"
 ARG JANUS_LIBSRTP_VERSION="2.3.0"
-ARG JANUS_USRSCTP_VERSION="547d3b46c64876c0336b9eef297fda58dbe1adaf"
-ARG JANUS_BORINGSSL_VERSION="8b601c88fb9b82b49407b018ff5e626a59600669"
-ARG JANUS_LIBWEBSOCKETS_VERSION="v4.0.20"
+ARG JANUS_USRSCTP_VERSION="0.9.5.0"
+ARG JANUS_BORINGSSL_VERSION="ac3f4fb8e1e3fd6da51033514e247e7924b16439"
```
Comment (christophe-kamphaus-jemmic) [inline Dockerfile]: Checking https://boringssl.googlesource.com/boringssl/+log/refs/heads/master we see that there are newer commits.
Why not use `225e8d39b50757af56e61cd0aa7958c56c487d54`?

Comment (jhon-freitez-jemmic) [reply Dockerfile]: Basically, I'm not taking commits after the Janus `v1.0.0` release date... But I can try latest versions as well.

File: Dockerfile (hunk 4 above)
Comment (jhon-freitez-jemmic) [reply Dockerfile]: I seems like there is an issue in the official documentation: https://janus.conf.meetecho.com/docs/DEPS.html

> rabbitmq-c: https://github.com/alanxz/rabbitmq-c (optional, v1.0.4, RabbitMQ)

I've fixed it already.

**Review by christophe-kamphaus-jemmic — APPROVED** [review]

**Pull request merged**